.github/workflows/gh-pages-deploy.yml

```diff
@@ -9,25 +9,26 @@ jobs:
   build-deploy:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - name: Setup Node.js 12.x
-      uses: actions/setup-node@v1
-      with:
-        node-version: '12.x'
-    - name: yarn install, yarn build
-      run: |
-        yarn --frozen-lockfile
-        yarn build
-        echo 'anj.aragon.org' > dist/CNAME
-      env:
-        NODE_ENV: 'production'
-        SITE_ROOT: 'https://anj.aragon.org/'
-        SENTRY_DSN: ${{ secrets.GH_PAGES_SENTRY_DSN }}
-        PORTIS_DAPP_ID: ${{ secrets.GH_PAGES_PORTIS_DAPP_ID }}
-        FORTMATIC_API_KEY: ${{ secrets.GH_PAGES_FORTMATIC_API_KEY }}
-    - name: push to gh-pages
-      uses: peaceiris/actions-gh-pages@v2
-      env:
-        ACTIONS_DEPLOY_KEY: ${{ secrets.GH_PAGES_ACTIONS_DEPLOY_KEY }}
-        PUBLISH_BRANCH: gh-pages
-        PUBLISH_DIR: ./dist
+      - uses: actions/checkout@v1
+      - name: Setup Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: yarn install, yarn build
+        run: |
+          yarn --frozen-lockfile
+          yarn build
+          echo 'anj.aragon.org' > dist/CNAME
+        env:
+          NODE_ENV: 'production'
+          SITE_ROOT: 'https://anj.aragon.org/'
+          SENTRY_DSN: ${{ secrets.GH_PAGES_SENTRY_DSN }}
+          PORTIS_DAPP_ID: ${{ secrets.GH_PAGES_PORTIS_DAPP_ID }}
+          FORTMATIC_API_KEY: ${{ secrets.GH_PAGES_FORTMATIC_API_KEY }}
+          COMMIT_SHA: GITHUB_SHA
```

Comment:
I haven’t tried this yet, let’s see if it works… it is needed because there is no `git` available in the GitHub Actions environment so we can’t do this: https://github.com/aragon/anj.aragon.org/blob/ad716f6996348f5ccd0b6d2e59168f39169076c1/next.config.js#L8